code/drasil-website/lib/Drasil/Website/Example.hs

```diff
@@ -79,16 +79,16 @@ fullExList codePth srsDoxPth = Enumeration $ Bullet $ map (, Nothing) (allExampl
 allExampleList :: [Example] -> [ItemType]
 allExampleList = map (\x -> Nested (nameAndDesc x) $ Bullet $ map (, Nothing) (individualExList x))
   where
-    nameAndDesc E{sysInfoE = SI{_sys = sys, _purpose = purp}} = S (programName sys) +:+ S " - To" +:+. head purp
+    nameAndDesc E{sysInfoE = SI{_sys = sys, _purpose = purp}} = S (abrv sys) +:+ S " - To" +:+. head purp
```

Comment:
Isn't this a regression? We want to use `abrv` less, and `programName` was a step in that direction.

Reply:
Isn't this reverting a regression instead? `abrv` carries the actual abbreviation of the `PD Controller` project. `programName` calculates what a 'valid program name' would be given the abbreviation (replacing a space with an underscore).

Reply:
Ok, for this one,  I think you're right.